src/slacko.ml

```diff
@@ -599,16 +599,16 @@ let optionally_add key value uri = match value with
 
 let definitely_add key value = optionally_add key (Some value)
 
-let unauthed_endpoint ~base_url e =
+let unauthed_endpoint ~base_url meth =
```

Comment:
Admittedly a bit pedantic, but I'd rather name it `method'` or `method_` since `meth` has kinda weird connocations.